### ISSUE TITLE
CRIS-Preisträger: Förderungs-Disclaimer

### ISSUE THREAD

**Issue by AndiWebred** [RRZE-Webteam/fau-cris#260]
Es geht um das Plugin für die Preisträger/innen.

Laut H3 müsste zumindest bei den ERC-Grants unter jedem Preisträger ein Disclaimer ergänzt werden, der daraufhinweist, wer das Forschungsprojekt fördert.

Ich würde deshalb vorschlagen wir ergänzen zwei neue Felder im CRIS-Backend, die je nach Bedarf dann mit dem Plugin ausgegeben werden.

Feld1: Logo (Bild-Upload) Hier kann das Logo des Förderers hochgeladen werden. Das ist im Falle der ERC-Grants immer das gleiche Logo-Bild. Vielleicht gibt es dafür dann eine Art Mediathek?
Feld2: Disclaimer: Hier muss der Disclaimer-Text rein. Der ist bei allen unterschiedlich, da jeweils die grant agreement No. mit angegeben werden muss. 

Beides würde ich dann im Wordpress-Frontend nebeneinander stellen wollen. Links das Logo rechts der Text?

**Comment by BastianMelsheimer** [comment]
@AndiWebred Soll dieses Issue reanimiert werden? Dann lass uns mal Kontakt aufnehmen?

**Comment by AndiWebred** [comment]
Ja, das ist bestimmt noch nötig. Lass uns gerne dazu austauschen.
